python/src/scipp/runtime_config.py

```diff
@@ -77,7 +77,12 @@
         "linewidth": [1.5],
         # Default line style for 1D non-histogram plots
         "linestyle": ["none"],
+        # Default padding around matplotlib axes
         "padding": [0.05, 0.02, 1, 1],
+        # Default monitor pixel ratio for 3d plots.
+        # Set to > 1 for high density (e.g. retina) displays.
+        "pixel_ratio":
```

Comment:
Document somewhere?

Reply:
Maybe I should document all the config options in the https://scipp.github.io/reference/runtime-configuration.html
Even though we do say "The fields in the configuration file are not strictly defined and may vary between Scipp versions. For this reason it is advised to browse this file to determine what options are available."
Maybe I should make that into a Warning box?